webgui/src/flipmouse/js/tabslot.js

```diff
@@ -35,13 +83,95 @@ tabSlot.initSlots = function () {
     });
 };
 
+tabSlot.initProcesses = function () {
+	if(inElectron === false) {
+		var resultList = ["not available / nicht verfügbar"];
```

Comment:
maybe instead add an id to `<div id="processSelectionDiv" class="row">` above and just use `L.setVisible('#processSelectionDiv', false)` to completely hide the whole section.